CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 10.0.2 (2024-04-16)
+
+- Add support for `@skyux/help-inline` and bump `ng2-dragula` version [#279](https://github.com/blackbaud/skyux-docs-tools/pull/280)
```

Comment:
By this description, I believe a minor version bump is warranted.

Reply:
I think I would personally just reword this into a “fix”. We aren’t adding new functionality. We are fixing the old functionality to pull in all appropriate things.

Reply:
That being said - I don’t care strongly either way. I would agree that the current wording sounds like a minor fix.

Reply:
I can change the wording to fix, and change it to a minor version bump. i am throwing a flex topic on abt this tho bc I'm curious what the conventions are for how we decide patch/minor/major

Reply:
Major = breaking change
Minor = new feature, addition
Patch = bug fix (generally)

Reply:
gotcha, i think this is more an addition that takes care of a bug we're noticing. This would've been added anyways, we just missed it so it feels like a "fix"

Reply:
https://semver.org/